doc/next/_sources/campaign-overview.rst.txt

```diff
@@ -0,0 +1,86 @@
+.. _campaign_overview:
+
+Overview
+========
+
+Campaigns allow you, the seller, to group and manage your ads together, having shared budgets, targeting rules, and other settings. You can create separate campaigns to run ads in different locations or using different budgets.
+
+Campaigns are a simpler way to organize and manage your products, services, and ads in general.
+They are becoming first-class citizen in Admarkt, and as such, are a primary container to get your advertisements online.
```

Comment:
This line will be irrelevant over time. Do we generally put these kind of roadmap/marketing messages in the API docs? same applies for line 11

Reply:
IMHO it's very relevant to point out that campaigns are not an optional thing anymore, but suddenly something you have to manage if you want your ads online. We can re-word it? It's not about marketing

Reply:
Campaigns are the standard for organising your ads, they will serve as the primary container to get your ads online. 

I am more so suprised we do product announcements/developments within the developer docs.

Reply:
Oke, will reword :) 
Where would we be better off announcing product changes? They don't happen too common, don't think we have a playbook